datatype.go

```diff
@@ -23,58 +25,86 @@ const (
 	binaryType
 	timeType
 	booleanType
+	// the following are not snowflake types per se but internal types
+	null
```

Comment:
Please change `null` to something more obvious name... I got confused because `null` looks like a Go native data type... Can this be `nullType` ?